grass7/imagery/i.landsat8.swlst/landsat8_mtl.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python\<nl>\
```

Comment:
Perhaps you can change this, while you're at it, to something that makes sense, like:
`#!/usr/bin/env python`

Reply:
Sure, done  (I found a few more which I have fixed as well in ee88869)